packages/mdc-ripple/README.md

```diff
@@ -182,6 +182,23 @@ ripple.
 MDCRipple.attachTo(document.querySelector('.surface'));
 ```
 
+### Ripple API
+
+The component allows for programmatic activation / deactivation of the ripple, for complex interaction between deeply
+tied components. This is used for making form field labels trigger the ripples in their corresponding input elements,
+for example.
+
+#### MDCRipple.activate()
+
+Triggers an activation of the ripple (the first stage, which happens on a down event). It is represented visually as if
```

Comment:
could we say "which happens when the ripple surface is engaged via interaction, such as a `mousedown` or a `pointerdown` event instead of "which happens on a down event"? Also I'm not sure we need to talk about the visual representation as coming from a keyboard event; we can just say it "expands from the center" or something a bit less implementation-specific instead.